multichain-testing/scripts/ymax-tool.ts

```diff
@@ -118,8 +142,41 @@ const main = async (
     connectWithSigner = SigningStargateClient.connectWithSigner,
   } = {},
 ) => {
-  const [volume] = argv.slice(2);
-  if (!volume) throw Error(`USAGE: ${argv[1]} 123.45`);
+  // Parse command line arguments using node:util's parseArgs
+  const { values, positionals } = parseArgs({
+    args: argv.slice(2),
+    options: {
+      'skip-poll': {
+        type: 'boolean',
+        default: false,
+      },
+      'exit-success': {
+        type: 'boolean',
+        default: false,
+      },
+      help: {
+        type: 'boolean',
+        short: 'h',
+        default: false,
+      },
```

Comment:
minor stylistic suggestion:

```suggestion
      'skip-poll': { type: 'boolean', default: false },
      'exit-success': { type: 'boolean', default: false },
      help: { type: 'boolean', short: 'h', default: false },
```

It annoys me when the formatter spreads code vertically so I can't see as much on the page.

Reply:
Maybe we should increase the value for `"printWidth"` in `.prettierrc`

Reply:
I don't think that would help. `prettier` isn't breaking the line because it's too long.